api/events/schema.py

```diff
@@ -12,11 +12,12 @@ class Meta:
 
     def check_status(self, info, **kwargs):
         try:
-            room_id = Room.query.filter_by(calendar_id=kwargs['calendar_id']).first().id  # noqa: E501
+            room_id = RoomModel.query.filter_by(
+                calendar_id=kwargs['calendar_id']).first().id
             if EventsModel.query.filter_by(
                     event_id=kwargs['event_id'],
                     room_id=room_id).count() > 0:
-                raise GraphQLError("This event already exists.")
+                raise GraphQLError("You cannot perform this action")
```

Comment:
Kindly explain which action is this that can't be performed

Reply:
The rationale behind this was that whenever someone checks in or cancels an event twice, they got the previous error message which was not as descriptive... Also in a case where someone has cancelled a meeting, they can't perform the checkin; which too was returning the previous message...Also not descriptive enough